isobus/src/isobus_virtual_terminal_objects.cpp

```diff
@@ -7,6 +7,8 @@
 /// @copyright 2023 The Open-Agriculture Developers
 //================================================================================================
 #include "isobus/isobus/isobus_virtual_terminal_objects.hpp"
+#include <iostream>
+#include <ostream>
```

Comment:
These headers appear unneeded. Maybe you had them in there for debugging? Can be removed I think.

Reply:
Yes, this is true, removed them.